### PR TITLE
man: add more info link and --locale example

### PR DESCRIPTION
**Pull request by patricedenis**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/man.md [hunk 1]
```diff
@@ -18,6 +19,10 @@
 
 `man -w {{command}}`
 
+- Display the man page for a custom overrided locale (L):
+
+`man {{command}} -L {{locale}}`
```
Comment (navarroaxel) [inline pages/common/man.md:24]: We prefer long argument format when is available:
```suggestion
`man {{command}} --locale={{locale}}`
```
(you can edit the other examples BTW)

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/man.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/man.md:24]: Should we have users perform this sort of update when they're just translating? @sbrl

Comment (navarroaxel) [reply pages/common/man.md:24]: :eyes: this isn't a translation PR. I didn't get it.

Comment (bl-ue) [reply pages/common/man.md:24]: Nevermind, I didn't even read this PR's title, I just assumed it was a translation 😂

Comment (patricedenis) [reply pages/common/man.md:24]: @bl-ue lol, I was just thinking may be I could also give other contribution than translation ^^
@navarroaxel : Ok for the long version of the options, I was not aware of the convention, my mistake

Comment (navarroaxel) [reply pages/common/man.md]: np @patricedenis, it's not explicit in the contribution guidelines, I think.

File: pages/common/man.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # man
 
 > Format and display manual pages.
+> More information: <https://www.man7.org/linux/man-pages/man1/man.1.html>
```
Comment (navarroaxel) [inline pages/common/man.md:4]:
```suggestion
> More information: <https://www.man7.org/linux/man-pages/man1/man.1.html>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/man.md [hunk 3]
```diff
@@ -18,6 +19,10 @@
 
 `man -w {{command}}`
 
+- Display the man page for a custom overrided locale (L):
```
Comment (navarroaxel) [inline pages/common/man.md:22]:
```suggestion
- Display the man page using a specific locale:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**